pkg/capabilities/capabilities.go

```diff
@@ -174,6 +174,10 @@ type TargetCapability interface {
 	CallbackCapability
 }
 
+// DON represents a network of connected nodes.
+//
+// For an example of an empty DON check, see the following link:
+// https://github.com/smartcontractkit/chainlink/blob/2a1b0c4eecb9cc23a364d62a080cc9ae0d08c518/core/capabilities/transmission/local_target_capability.go#L31
```

Comment:
This check is actually incorrect because we changed the type from string to int. Maybe just say here that ID==0 means invalid/empty DON?